js/index.js

```diff
@@ -4,16 +4,18 @@ var search = document.getElementById("search");
 
 search.addEventListener("click", () => {
     var searchText = document.getElementById("searchText").value;
-    url = `https://www.googleapis.com/books/v1/volumes?q=${searchText}&maxResults=40`;
+    url = `https://www.googleapis.com/books/v1/volumes?q=${searchText}&download=pdf&maxResults=40`;
     if (searchText !== "" && searchText.trim(" ").length !== 0) {
         getData(url);
+        
     } else {
         window.alert("Please enter a search term that contains at least one alphanumeric character")
     }
+    searchText = "";
 })
 
 window.onload = () => {
-    url = "https://www.googleapis.com/books/v1/volumes?q=pride+prejudice&maxResults=40"
+    url = "https://www.googleapis.com/books/v1/volumes?q=pride+prejudice&download=epub&maxResults=40"
```

Comment:
You are repeating this URL. Declare it in the global scope and reuse it

Reply:
one of them takes an input parameter, and the other doesn't. And when declare the URL globally, and the input parameter in the functions, I get undefined.

Reply:
Done. had to use a function however